src/common/components/repository-row/index.js

```diff
@@ -1,46 +1,46 @@
 import React, { PropTypes } from 'react';
 
-import Button from '../vanilla/button';
-
-import styles from './repositoryRow.css';
+import { TR, TD } from '../vanilla/table';
+import { parseGitHubRepoUrl } from '../../helpers/github-url';
 
 const RepositoryRow = (props) => {
 
   const {
-    errorMsg,
-    repository,
-    buttonLabel,
-    buttonDisabled,
-    onButtonClick
+    snap
   } = props;
 
+  const { fullName } = parseGitHubRepoUrl(snap.git_repository_url);
+
+  return (
+    <TR>
+      <TD><a href={ `/${fullName}/builds` }>{ fullName }</a></TD>
+      <TD></TD>
+      <TD>{ renderSnapNameLabel(snap) }</TD>
+      <TD></TD>
```

Comment:
This would currently require calling the `/api/launchpad/builds` route once per repository and picking out the most recent one.

At present, this will be very inefficient.  I propose that we omit this column for MWC until we have a chance to work out how to get this information efficiently (perhaps by augmenting the Launchpad `Snap` class with an attribute representing the latest build date which we could populate in bulk from the query used via `/api/launchpad/snaps/list`).